test/spec/modules/userId_spec.js

```diff
@@ -1082,14 +1082,22 @@ describe('User ID', function() {
     it('callback for submodules that always need to refresh stored id', function(done) {
       let adUnits = [getAdUnitMock()];
       let innerAdUnits;
-      const parrableStoredId = '01.1111111111.test-eid';
-      const parrableRefreshedId = '02.2222222222.test-eid';
-      utils.setCookie('_parrable_eid', parrableStoredId, (new Date(Date.now() + 5000).toUTCString()));
+      const parrableStoredId = {
+        eid: '01.1111111111.test-eid',
+        ibaOptout: false,
+        ccpaOptout: false
+      };
+      const parrableRefreshedId = {
+        eid: '02.2222222222.test-eid',
+        ibaOptout: false,
+        ccpaOptout: false
+      };
+      utils.setCookie('_parrable_id', JSON.stringify(parrableStoredId), (new Date(Date.now() + 5000).toUTCString()));
```

Comment:
to remain consistent with our we encode/decode the compound cookie, shouldn't we be serializing the object instead of JSON stringifying it, even in this test?